test/confit-test.js

```diff
@@ -26,6 +26,10 @@ test('confit', function (t) {
 
 
     t.test('get', function (t) {
+        //setting process.env.env to development, should not change 'env:env'.
+        //This should be ignored and 'env:env' should solely depend on process.env.NODE_ENV
+        process.env.env = 'development';
```

Comment:
Essentially the new test should fail with the old code. I haven't run this locally but it _appears_ that the change to the test would have passed with both old and new code.